packages/core/src/scripts/tests/migrations.test.ts

```diff
@@ -429,11 +429,11 @@ describe('useMigrations: true', () => {
     const tmp = await testdir({
       ...symlinkKeystoneDeps,
       'app.db': await fs.readFile(`${prevCwd}/app.db`),
-      'keystone.js': await fs.readFile(`${__dirname}/fixtures/one-field-with-migrations.ts`),
+      'keystone.js': await fs.readFile(`${__dirname}/fixtures/one-field-with-migrations-2.ts`),
     });
     const recording = recordConsole({
       'Do you want to continue? All data will be lost.': true,
-      'Name of migration': 'init',
+      'Name of migration': 'init2',
```

Comment:
```
{
  "oldMigrationName": "20230504081137_init",
  "migrationName": "20230504081138_init2"
}
```

This was previously failing when the timestamps were the same, and hence the migration names were the same